src/App.vue

```diff
@@ -141,6 +144,27 @@ export default {
       await Promise.all(Object.entries(tokens).map(async ([token]) => this
         .addTokenBalance({ token, balance: await tokenBalance(token, address) })));
     },
+    async getCookiesList(address) {
+      await this.backendAuth('getCookiesList', address)
+        .then((list) => list.forEach((entry) => this.setCookiesListEntry(entry)));
+    },
+    // eslint-disable-next-line consistent-return
+    async backendAuth(method, address) {
+      const { challenge } = await Backend[method](address);
+      if (this.useSdkWallet) {
+        const signature = await client.signMessage(challenge);
+        const response = await Backend[method](address, { challenge, signature });
+        return response;
+      }
+
+      const url = new URL(window.location, window.location);
+      url.search = '';
+      window.location = createDeepLinkUrl({
```

Comment:
I can't find how the returned cookies list is handled if the web wallet used by deeplinks

Reply:
@davidyuk I need help here how to get the response using deeplink

Reply:
There is no easy way, so let's don't sync cookies settings if sdk wallet is not available